titus-api/src/main/java/com/netflix/titus/api/jobmanager/JobConstraints.java

```diff
@@ -0,0 +1,84 @@
+/*
+ * Copyright 2020 Netflix, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.netflix.titus.api.jobmanager;
+
+import java.util.Set;
+
+import static com.netflix.titus.common.util.CollectionsExt.asSet;
+
+/**
+ * Constants and functions for job soft and hard constraints. Constraint names are case insensitive.
```

Comment:
it seems `SchedulingConstaintValidator` assumes they are always lowercase

Reply:
We ignore case, so it is ok for a user to pass for example `ExclusiveHost`